packages/palette/src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx

```diff
@@ -23,13 +21,27 @@ export const StaticCountdownTimer: React.SFC<{
   countdownEnd: string
   currentTime: string
 }> = ({ action, note, countdownEnd, countdownStart, currentTime }) => {
-  const actionDeadline = moment(countdownEnd)
-    .tz(moment.tz.guess())
-    .format("MMM DD, h:mm A z")
+  const dateTime = DateTime.fromISO(countdownEnd).toLocal()
+  const minutes = dateTime.minute < 10 ? "0" + dateTime.minute : dateTime.minute
+  const amPm = dateTime.hour >= 12 ? "pm" : "am"
+  let hour
+  if (dateTime.hour > 12) {
+    hour = dateTime.hour - 12
+  } else if (dateTime.hour === 0) {
+    hour = 12
+  } else {
+    hour = dateTime.hour
+  }
+  const time = `${hour}:${minutes}${amPm}`
```

Comment:
Have you looked into using Luxon's `toFormat` function to format the time object?

https://github.com/moment/luxon/blob/master/docs/formatting.md#toformat

Reply:
Oh I hadn't seen this, thanks!

Reply:
@dblandin Opting for how I had written it in the existing code as there's a bit more control, this can be helpful in the future when we don't need such specific styling requirements.